.github/workflows/release-test.yml

```diff
@@ -0,0 +1,201 @@
+        if [ -z "$DOCKER_VERSION" ]; then
+          DOCKER_VERSION="latest"
+        fi
+        docker pull riot/riotbuild:$DOCKER_VERSION
+    - name: Create TAP interfaces
+      run: |
+        sudo RIOT/dist/tools/tapsetup/tapsetup -c 11
+    - name: Run release tests
+      timeout-minutes: 350
+      run: |
+        RIOTBASE="$GITHUB_WORKSPACE/RIOT"
+        TOX_ARGS=""
+        if ! echo ${{ github.event.inputs.riot_version }} | \
+              grep -q "[0-9]\{4\}.[0-9]\{2\}-RC[0-9]\+"; then
+          TOX_ARGS+="--non-RC "
+        fi
+        CORES=$(cat /proc/cpuinfo | grep 'processor\s*:\s*[0-9]\+' | wc -l)
+        DOCKER_MAKE_ARGS="-j${CORES}"
+
+        cd Release-Specs
+        # definition in env does not work since $GITHUB_WORKSPACE seems not to
+        # be accessible
+        sudo \
+          BUILD_IN_DOCKER=1 \
+          DOCKER_MAKE_ARGS=${DOCKER_MAKE_ARGS} \
+          DOCKER_ENV_VARS=USEMODULE \
+          GITHUB_REPOSITORY=${GITHUB_REPOSITORY} \
+          GITHUB_RUN_ID=${GITHUB_RUN_ID} \
+          GITHUB_SERVER_URL=${GITHUB_SERVER_URL} \
+          RIOTBASE=${RIOTBASE} \
+          $(which tox) -- ${TOX_ARGS} -m "not iotlab_creds"
+    - name: junit2html and XML deploy
+      if: ${{ always() }}
+      run: |
+        DATE=$(date +"%Y-%m-%d-%H-%M-%S")
+        if echo ${{ github.event.inputs.riot_version }} | \
+              grep -q "[0-9]\{4\}.[0-9]\{2\}"; then
+          VER=${{ github.event.inputs.riot_version }}
+        else
+          VER=$(git -C $GITHUB_WORKSPACE/RIOT rev-parse --short HEAD)
+        fi
+        mkdir test-reports/
+        junit2html $GITHUB_WORKSPACE/Release-Specs/test-report.xml \
+          test-reports/test-report-native-$VER-$DATE.html
+        cp $GITHUB_WORKSPACE/Release-Specs/test-report.xml \
+          test-reports/test-report-native-$VER-$DATE.xml
+    - uses: actions/upload-artifact@v2
+      if: ${{ always() }}
+      with:
+        name: Upload test report
+        path: test-reports/*
+  iotlab-tasks:
+    runs-on: ubuntu-latest
+    timeout-minutes: 360
+    steps:
+    - name: Generate .riotgithubtoken
+      run: |
+        echo '${{ secrets.RIOT_CI_ACCESS_TOKEN }}' > ~/.riotgithubtoken
+    - name: Checkout Release-Specs
+      uses: actions/checkout@v2
+      with:
+        # TODO: change to RIOT-OS/Release-Specs
+        repository: miri64/Release-Specs
+        path: Release-Specs
+        fetch-depth: 1
+        ref: ${{ github.event.inputs.release_specs_version }}
+    - name: Checkout RIOT
+      uses: actions/checkout@v2
+      with:
+        repository: RIOT-OS/RIOT
+        path: RIOT
+        fetch-depth: 1
+        ref: ${{ github.event.inputs.riot_version }}
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.x
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox tox-gh-actions junit2html
```

Comment:
```suggestion
        python -m pip install tox tox-gh-actions junit2html
```

Reply:
Why?

Reply:
It's consistent with the line above and it ensures that the same pip is used when installing the dependencies.

Reply:
I'm just using what the github-actions doc provides: https://docs.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#starting-with-the-python-workflow-template

Reply:
I insist: the line above, `pip` is updated using the pip installed with the system wide Python version. Here you use pip without specifying which one to use, so it could be an other version (I already had this issue in another project). Using `python -m pip` for both ensures the same pip is updated and used after to install the dependencies.

Reply:
Fine. If you made the experience already then I will adapt it. But maybe you should also inform GitHub / [actions/python-setup](https://github.com/actions/setup-python) then as well, that their documentation is wrong.